pkg/workflow/controller/handlers/pod/gc.go

```diff
@@ -24,6 +24,20 @@ func IsGCPod(pod *corev1.Pod) bool {
 	return true
 }
 
+// IsWorkloadPod judges whether a pod is a workload pod by check whether it has
+// annotation "workload.cyclone.dev".
```

Comment:
annotation "workload.cyclone.dev"? Maybe it should be label "pod.kubernetes.io/kind".

Reply:
Sorry, It is a label, but the code comments are not correct, I will address it.

Reply:
PTAL